ci/docs/build.sh

```diff
@@ -44,18 +44,18 @@ conda list --show-channel-urls
 
 # Build Python docs
 gpuci_logger "Build Sphinx docs"
-cd $PROJECT_WORKSPACE/docs
+cd "$PROJECT_WORKSPACE/docs"
 make html
 
 #Commit to Website
-cd $DOCS_WORKSPACE
+cd "$DOCS_WORKSPACE"
 
 for PROJECT in ${PROJECTS[@]}; do
     if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
         mkdir -p api/$PROJECT/$BRANCH_VERSION
     fi
-    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
+    rm -rf "$DOCS_WORKSPACE/api/"$PROJECT/$BRANCH_VERSION/*	
 done
 
-mv $PROJECT_WORKSPACE/docs/build/html/* $DOCS_WORKSPACE/api/cuspatial/$BRANCH_VERSION
+mv "$PROJECT_WORKSPACE/docs/build/html/"* "$DOCS_WORKSPACE/api/cuspatial/"$BRANCH_VERSION
```

Comment:
```suggestion
mv "$PROJECT_WORKSPACE/docs/build/html/"* "$DOCS_WORKSPACE/api/cuspatial/$BRANCH_VERSION"
```

I don't think there's any harm in moving this to the end of the line